sdv/data_navigator.py

```diff
@@ -147,7 +143,8 @@ def _anonymize_data(self):
 
             pii_fields = self.ht._get_pii_fields(ht_meta)
             if pii_fields:
-                table.data = ht_table
+                # Table is a namedtuple, which is immutable, so instantiate a new one with transformed data
```

Comment:
This line is too long and makes the TravisCI build crash, but a part from this, all your changes seems good to merge, could you please shorten it?

Reply:
Also a tip: If you want to check for linting issues on your local environment before open the PR, you can do it with `make lint`, or run the same build that TravisCI does with `make test-all`. In case you want further reference, please find our Contributing Guide [here](https://github.com/HDI-Project/SDV/blob/master/CONTRIBUTING.rst)

Reply:
Thanks for the heads up - I found the contributing guide after I had used travis to debug. 😄